src/app/screens/Home/index.tsx

```diff
@@ -94,6 +100,33 @@ function Home() {
     }
   }
 
+  async function onTabChangeHandler(index: number) {
+    if (index === 1) {
+      await loadInvoices();
```

Comment:
should we add a check there that this is only done if the invoices are NOT yet loaded?

Right now it always reloads when I change the tab